### PR TITLE
create staging tables for user events

### PR DESCRIPTION
**Pull request by agpmercado**
staging tables used for customer.io migration

### CODE REVIEW

File: models/staging/portal/stg_portal_exit_survey_reason_user [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+renamed as ( 
+
+    select
+--ids
+        id as cancel_id,
+        user_id,
+        reason_id,
+
+-- timestamps
+        case when timestamp(reactivated_date) < cancelled_date then DATETIME(TIMESTAMP_ADD(cancelled_date, INTERVAL 1 MINUTE), 'Australia/Sydney') else DATETIME(timestamp(reactivated_date), 'Australia/Sydney') end as reactivate_datetime, --capture reactivate_date < cancelled_date until datawash is done
```
Comment (alewinskylyka) [inline models/staging/portal/stg_portal_exit_survey_reason_user:16]: @agpmercado can we reformat case when like [this](https://github.com/mattm/sql-style-guide#aligning-casewhen-statements:~:text=Aligning%20case/when%20statements)

I like the first example.

**Review by alewinskylyka — CHANGES_REQUESTED** [review]
Hi Ana, looks good! Can we just reformat the case when for easy reading.

File: models/staging/portal/stg_portal_exit_survey_reason_user (hunk 1 above)
Comment (agpmercado) [reply models/staging/portal/stg_portal_exit_survey_reason_user:16]: done

File: models/staging/portal/stg_portal_exit_survey_reason_user [hunk 2]
```diff
@@ -13,7 +13,10 @@ renamed as (
         reason_id,
 
 -- timestamps
-        case when timestamp(reactivated_date) < cancelled_date then DATETIME(TIMESTAMP_ADD(cancelled_date, INTERVAL 1 MINUTE), 'Australia/Sydney') else DATETIME(timestamp(reactivated_date), 'Australia/Sydney') end as reactivate_datetime, --capture reactivate_date < cancelled_date until datawash is done
+        case 
+            when timestamp(reactivated_date) < cancelled_date then DATETIME(TIMESTAMP_ADD(cancelled_date, INTERVAL 1 MINUTE), 'Australia/Sydney')
```
Comment (alewinskylyka) [inline models/staging/portal/stg_portal_exit_survey_reason_user:17]: sorry Im being really pedantic but can we have the then statement on a separate line?

**Review by alewinskylyka — CHANGES_REQUESTED** [review]

File: models/staging/portal/stg_portal_exit_survey_reason_user (hunk 2 above)
Comment (agpmercado) [reply models/staging/portal/stg_portal_exit_survey_reason_user:17]: I thought you meant first example on the link above? the first example had it on the same line!
so do we want to have when & then in separate lines? so example 2?

i.e. below

-- Good too
select
    case
        when event_name = 'viewed_homepage'
            then 'Homepage'
        when event_name = 'viewed_editor'
            then 'Editor'
        else 'Other'            
    end as page_name
from events

Comment (agpmercado) [reply models/staging/portal/stg_portal_exit_survey_reason_user:17]: oops that lost the indentation

Comment (alewinskylyka) [reply models/staging/portal/stg_portal_exit_survey_reason_user:17]: oh yes sorry i meant the second version! I prefer 'then' on a different line but dont mind about the indentation.
I would prefer it like you have above without the indentation but no biggie.

**Comment by alewinskylyka** [comment]
Looks good! Now I can do the weld migration

**Pull request merged**